### PR TITLE
Feature/5 syntax errors

### PR DESCRIPTION
**Pull request by BaranovDaniil**
Added selection of a line where an error occurred and message for it.
image

### CODE REVIEW

File: lib/language_syntax/golang_syntax.dart [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      continue;
+    }
+
+    if (currentLine.startsWith(RegExp("\\s*/\\*"))) {
+      while ((!currentLine.contains(RegExp("\\*/\\s*"))) &&
+          (i < lines.length - 1)) {
+        i++;
+        currentLine = lines[i];
+      }
+    } else if (currentLine.startsWith(RegExp(".*`"))) {
```
Comment (SvetlanaRyzhkova) [inline lib/language_syntax/golang_syntax.dart]: move else/else if   to next line everywhere

File: lib/language_syntax/golang_syntax.dart [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      } while ((!currentLine.contains(RegExp("`"))) && (i < lines.length - 1));
+    }
+
+    if (currentLine.trim().endsWith("}")) {
+      indentLevel--;
+    }
+
+    for (int countOfSpace = 0;
+        countOfSpace < currentLine.length;
+        countOfSpace++) {
```
Comment (SvetlanaRyzhkova) [inline lib/language_syntax/golang_syntax.dart]: I think that the design for should be all in one line, if it does not go beyond the screen size.

File: lib/language_syntax/java_dart_syntax.dart [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// Обработка точек с запятой для цикла for,
+// без учета комментариев и мультистрок.
```
Comment (SvetlanaRyzhkova) [inline lib/language_syntax/java_dart_syntax.dart]: An error pops up if it is incorrect to write the for design in quotes?

**Review by SvetlanaRyzhkova — COMMENTED** [review]
I think we should not commit the example files and pubspec.lock files

File: lib/language_syntax/golang_syntax.dart [hunk 4]
```diff
@@ -0,0 +1,57 @@
+// Обработка правильности отступов, точек с запятой для цикла for
+// с учетом комментариев, с учетом строк.
```
Comment (GrigoriyKostryukov) [inline lib/language_syntax/golang_syntax.dart]: Please, translate comments that are not in English

**Review by GrigoriyKostryukov — CHANGES_REQUESTED** [review]

File: lib/language_syntax/golang_syntax.dart (hunk 1 above)
Comment (BaranovDaniil) [reply lib/language_syntax/golang_syntax.dart]: I don't do that, but the code reformer built into android studio.

File: lib/language_syntax/golang_syntax.dart (hunk 2 above)
Comment (BaranovDaniil) [reply lib/language_syntax/golang_syntax.dart]: I don't do that, but the code reformer built into android studio.

File: lib/language_syntax/golang_syntax.dart (hunk 4 above)
Comment (BaranovDaniil) [reply lib/language_syntax/golang_syntax.dart]: done

File: lib/language_syntax/golang_syntax.dart (hunk 2 above)
Comment (a-kokorev) [reply lib/language_syntax/golang_syntax.dart]: I see that there's exactly similar line in `java_dart_syntax.dart`, that wasn't formatted like that. And I do not see any reason for dividing that expression into several lines

File: lib/language_syntax/golang_syntax.dart (hunk 1 above)
Comment (a-kokorev) [reply lib/language_syntax/golang_syntax.dart]: I don't mind having it this way. Just make sure that the style is the same throughout the project

**Review by GrigoriyKostryukov — APPROVED** [review]

**Review by darki-D4C — APPROVED** [review]
Could you please delete changes to pubspec.lock in commit?
Anyhow - LGTM!

**Review by a-kokorev — APPROVED** [review]

**Pull request merged**